### DISCUSSION TITLE
Unintended vulgar behavior?

### DISCUSSION THREAD

**Discussion by saleguas** [community/community#10787] [Copilot Conversations]
Not sure if this was intended behavior, but I could make Copilot same some pretty vulgar stuff if I "steered" it into the right direction enough.
![image](https://user-images.githubusercontent.com/42396136/151723926-111be561-556f-40c9-a53b-fb69ed5eb06a.png)

**Comment by dave9123** [comment]
Even when I typed in javascript
`// Open youtube.com`
It goes a rick roll video 😂
